### DISCUSSION TITLE
SEGYSAK sprint - '#t21-hack-segysak'

### DISCUSSION THREAD

**Discussion by trhallam** [softwareunderground/transform-2021-hackathon#11] [Projects]
Possibility for people to help out again on a SEGYSAK sprint.

There are a long list of things to trial, test and document so check out the [Kanban board](https://github.com/trhallam/segysak/projects/3) and see if something strikes your fancy. Otherwise, propose something new or a feature you really want!

We also have plenty of space for examples and for documentation review so if you have something you want to share that is also most welcome. Examples are generally written as notebooks before being converted via `jupytxt` to `.py` files.

To get contributing, join on the hack channel on Swung and someone will get you started.

**Comment by filippo82** [comment]
[T21 Hackathon Idea Mashup](https://softwareunderground.org/events/2021/4/7/t21-hackathon-idea-mashup) today (or tomorrow depending on your time zone) Wednesday April 14th at 16:00 UTC. Come to Slack to find the link for the meeting!
